src/coreclr/pal/src/init/pal.cpp

```diff
@@ -580,7 +581,7 @@ Initialize(
         if (flags & PAL_INITIALIZE_FLUSH_PROCESS_WRITE_BUFFERS)
         {
             // Initialize before first thread is created for faster load on Linux
-            if (!InitializeFlushProcessWriteBuffers())
+            if (!minipal_initialize_flush_process_write_buffers())
```

Comment:
I think `minipal_initialize_flush_process_write_buffers` needs to be called from Mono initialization, standalone GC initialization and/or NativeAOT initialization as well.

Reply:
Sounds reasonable to me.